lib/pf/util.pm

```diff
@@ -296,27 +296,27 @@ our $NON_VALID_MAC_REGEX = qr/^(00|ff)(:\g1){5}$/;
 our $VALID_PF_MAC_REGEX = qr/^[0-9a-f]{2}(:[0-9a-f]{2}){5}$/;
 
 sub valid_mac {
```

Comment:
since this is such a critical function, @jrouzierinverse, can you take a look at this too just to make sure

Reply:
This is just a cleanup of whitespace and using a global IP address regex it changes no functionality.

Reply:
yes, agreed, just wanted to have at least 2 pair of eyes look at it